src/MinFraud/ReportTransaction.php

```diff
@@ -46,9 +46,11 @@ public function __construct(
      * @param array  $values         An array of transaction parameters. The keys are the same
      *                               as the JSON keys. You may use either this or the named
      *                               arguments, but not both.
-     * @param string $ipAddress      Required. The IP address of the customer placing the
+     * @param string $ipAddress      Optional. The IP address of the customer placing the
      *                               order. This should be passed as a string like
-     *                               "44.55.66.77" or "2001:db8::2:1".
+     *                               "44.55.66.77" or "2001:db8::2:1". This field is not
```

Comment:
Also on me, but these "field" uses probably should all be "argument". It might be better for us to just do that in a separate PR though as I suspect it is endemic.